css/BEM-style.md

```diff
@@ -0,0 +1,223 @@
+    display:flex;
+    flex-direction:column;
+    justify-content: center;
+    align-items: flex-start;
+    
+    &__text{
+       margin:2rem;
+    } 
+    &__buttons{
+        margin:0.5rem;   
+    }  
+}
+.btn{
+    text-decoration: none;
+    border:1px solid saddlebrown;
+    padding:1rem 2rem;
+    color:white;
+    border-radius:113rem;
+    font-size:2rem;
+    text-align: center;
+    display:inline-block;
+    margin:2rem;
+    &--active{
+         background:saddlebrown;
+         &:hover{
+             background-color:rgb(201, 93, 17);
+         }
+    }
+    &--ghosted{
+        background:transparent;
+        &:hover{
+            background-color:rgb(201, 93, 17);
+        }
+    }
+}
+.primary-heading{
+    font-size:6rem;
+}
+```
+
+**3.How BEM is useful on larger codebases and scaling and writing maintainnable code:** 
+
+What if I told you to add a new button with some animation on it in the hero-box section? What other information will you need? Well nothing, since all necessary information is already present with you now and you know exactly where this component must lie and how you will name the classes on it. 
+This process of naming modifiers,elements and blocks might seem tedious at first, but as your project will grow larger and larger you'll realise how easy it has become to find and change the styles on a component or add styles to new components created. It made your HTML and CSS much more readable since the names of classes now make sense. You no longer need to worry about the specificity now and find out which particular class/id/tag is conflicting your newly written style.
+
+
+### Independent Practice
+
+Since you can use BEM pretty much anywhere , how about refactoring your previously written code for a project (very old projects). This will show you how your previously named classes, now makes it difficult for you to understand and maintain CSS code, but using BEM now will no longer create that issue for you. 
+
+### Challenge
+
+Try to use the BEM modelling on the following HTML snippet, which creates a simple navbar.
+```
+<header>
+    <nav>
+        <div>
+            <img src="" alt="Logo">
+        </div>
+        <div>
+            <li>
+                <a href="">Nav Link1</a>
+            </li>
+            <li>
+               <a href="">Nav Link2</a>
+            </li>
+            <li>
+                <a href="">Nav Link3</a>
+            </li>
+        </div>
+     </nav>
+ </header>
+```
+
+
+### Check for Understanding  
+
+Form small groups and discuss:
+
+- Why using a naming architecture/modelling principle like BEM is important?  
+- How BEM naming model can be used (the priciple for naming blocks,elements, and modifier)?  
+- How BEM avoids specificity conflicts?
+
+### Supplemental Materials
+BEM is not the only modelling architecture followed to maintain CSS files. There exists quite a few of them. You can read about them below:
+- [SMACSS]("http://smacss.com/)(5 min read)
```

Comment:
the quotes should not be with the URL - they are not linked right now.

Also, I suspect sitepoint article has the wrong URL linked - it leads to an ORM wiki.